CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/renderables/CVKAxesRenderable.java

```diff
@@ -622,10 +800,12 @@ public static int LoadShaders(CVKDevice cvkDevice) {
         int ret = VK_SUCCESS;
 
         try{
-            vertShaderSPIRV = compileShaderFile(CVKShaderPlaceHolder.class, "17_shader_vertexbuffer.vert", VERTEX_SHADER);
-            fragShaderSPIRV = compileShaderFile(CVKShaderPlaceHolder.class, "17_shader_vertexbuffer.frag", FRAGMENT_SHADER);
-
+            vertShaderSPIRV = compileShaderFile(CVKShaderPlaceHolder.class, "PassThru.vs", VERTEX_SHADER);
```

Comment:
Wait....we're compiling shaders from disk every frame????  It's good to have that code path but it should only get called if the .spv is missing or out of date.

Reply:
It's called once at the beginning of the graph creation